digitaltwins/samples/device/javascript/sample_device.js

```diff
@@ -28,10 +28,10 @@ const commandHandler = (request, response) => {
 };
 
 
-const environmentalSensor = new EnvironmentalSensor('environmentalSensor', propertyUpdateHandler, commandHandler);
+const environmentalSensor = new EnvironmentalSensor('sensor', propertyUpdateHandler, commandHandler);
```

Comment:
sensor seems vague. Why is it just sensor?

Reply:
... from the DTDL model